server/src/main/java/edu/jhu/server/data/Hallway.java

```diff
@@ -0,0 +1,35 @@
+package edu.jhu.server.data;
+
+public class Hallway implements ILocation {
```

Comment:
I was thinking the board would be a 5x5 array of Rooms. Hallways are their own enum, and there's another enum for INVALID. I don't like having a bunch of public static members